cli/integration/tests/waiter/test_cli_multi_cluster.py

```diff
@@ -58,3 +58,30 @@ def test_federated_show(self):
                     util.delete_token(self.waiter_url_2, token_name)
         finally:
             util.delete_token(self.waiter_url_1, token_name)
+
+    def test_federated_delete(self):
```

Comment:
Should we add a test for delete on a single cluster?

Reply:
Done, add `test_delete_single_cluster`